Neos.Flow/Classes/Persistence/Doctrine/Service.php

```diff
@@ -201,7 +201,7 @@ function ($message) use ($that) {
         }
 
         $configuration = new Configuration($connection, $outputWriter);
-        $configuration->setMigrationsNamespace(\Neos\Flow\Persistence\Doctrine\Migrations::class);
```

Comment:
IMHO this should be really a string to begin with. It a namespace, not a `::class`. And even `Migrations::class` does not resolve.